skimage/exposure/exposure.py

```diff
@@ -406,7 +412,7 @@ def rescale_intensity(image, in_range='image', out_range='dtype'):
                                             clip_negative=(imin >= 0)))
 
     # Fast test for multiple values, operations with at least 1 NaN return NaN
-    if np.isnan(imin + imax + omin + omax):
+    if np.any(np.isnan([imin, imax, omin, omax])):
```

Comment:
Why did you make this change?

Reply:
I’m not too fussed honestly but if you make the change you should remove the comment above it. =)

Reply:
True that the change is unrelated to the issue this PR fixes, but I thought it might be too small for an issue by itself. My problem with the old line was that it could lead to an overflow warning (commented in the commit message). As these are just four scalars there shouldn't be a large cost to creating the array.

You're absolutely right that the comment above it should be removed :) I guess there's no need for an additional explanation of what the line does, as the warning below explains it. Thanks for pointing that out!